planner/core/integration_test.go

```diff
@@ -4707,6 +4707,26 @@ func (s *testIntegrationSuite) TestIssue27797(c *C) {
 	result.Check(testkit.Rows("<nil>"))
 }
 
+func (s *testIntegrationSuite) TestIssue27949(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t27949")
+	tk.MustExec("create table t27949 (a int, b int, key(b))")
+	tk.MustQuery("explain select * from t27949 where b=1").Check(testkit.Rows("IndexLookUp_10 10.00 root  ",
```

Comment:
Maybe we can use `explain format = 'brief'` to test. And can use `select @@last_plan_from_binding` to check whether the plan use the binding.